nautobot_chatops/workers/nautobot.py

```diff
@@ -999,3 +1025,150 @@ def about(dispatcher, *args):
+
+        # Prompt user to select a circuit provider from a list of provider_options
+        dispatcher.prompt_from_menu(
+            "nautobot get-circuit-connections",  # command sub-command
+            "Select a circuit provider",  # Prompt to user
+            provider_options,  # Options to choose from
+            offset=menu_offset_value(provider_slug),
+        )
+        return False  # command did not run to completion and therefore should not be logged
+
+    try:
+        provider = Provider.objects.get(slug=provider_slug)
+    except Provider.DoesNotExist:  # If provider cannot be found, return STATUS_FAILED with msg
+        provider_not_found_error_msg = f"Circuit provider with slug {provider_slug} does not exist"
+        dispatcher.send_error(provider_not_found_error_msg)
+        return (CommandStatusChoices.STATUS_FAILED, provider_not_found_error_msg)
+
+    if menu_item_check(circuit_id):
+        circuit_options = [
+            (circuit.cid, circuit.cid) for circuit in Circuit.objects.filter(provider__slug=provider.slug)
+        ]
+        if not circuit_options:
+            no_circuits_found_error_msg = f"No circuits with provider slug {provider.slug} were found"
+            dispatcher.send_error(no_circuits_found_error_msg)
+            return (CommandStatusChoices.STATUS_SUCCEEDED, no_circuits_found_error_msg)
+        dispatcher.prompt_from_menu(
+            f"nautobot get-circuit-connections {provider_slug}",
+            "Select a circuit",
+            circuit_options,
+            offset=menu_offset_value(circuit_id),
+        )
+        return False  # command did not run to completion and therefore should not be logged
+
+    try:
+        circuit = Circuit.objects.get(cid=circuit_id)
+    except Circuit.DoesNotExist:
+        cid_not_found_msg = f"Circuit with circuit ID {circuit_id} not found"
+        dispatcher.send_error(cid_not_found_msg)
+        return (CommandStatusChoices.STATUS_FAILED, cid_not_found_msg)
+
+    if menu_item_check(circuit_id):
+        prompt_for_circuit(
+            "nautobot get-circuit-connections",
+            "Get Nautobot Circuit Connections",
+            dispatcher,
+            offset=menu_offset_value(circuit_id),
+        )
+        return False  # command did not run to completion and therefore should not be logged
+
+    # Ensure the termination endpoints are present, otherwise set to a string value
+    endpoint_info_a, endpoint_info_z = examine_termination_endpoints(circuit)
+
+    dispatcher.send_blocks(
+        dispatcher.command_response_header(
+            "nautobot",
+            "get-circuit-connections",
+            [("Provider Name", provider.slug), ("Circuit ID", circuit.cid)],
+            "circuit connection info",
+            nautobot_logo(dispatcher),
+        )
+    )
+
+    header = ["Side", "Connecting Object"]
+    rows = [("A", endpoint_info_a), ("Z", endpoint_info_z)]
+
+    dispatcher.send_large_table(header, rows)
```

Comment:
Given that there are only two rows here, use of send_large_table is a bit overkill, and I think not the most visually appealing way to present this information. Perhaps consider either:
1) Adding these directly to the command_response_header (they're not technically "args", but they are key-value pairs that you could append after the Provider Name and Circuit ID, and it might be more attractive that way)
2) Just sending these directly as a block of markdown text (`f"Side A: {endpoint_info_a}"`, `f"Side Z: {endpoint_info_z}"`)

Reply:
Hi Glenn.  I tried both your suggestions.
1. Adding these directly to the command_response_header gives a bad shortcut output
![image](https://user-images.githubusercontent.com/13647153/133514176-d673ce8e-57e5-45ed-a339-c69728a0980a.png)
2. Doing the straight markdown was OK, but I think it was harder to read.  I actually prefer to see the results in a table (even though it's not a large table) because of its column format

Reply:
Understood. Thanks for indulging me!